### DISCUSSION TITLE
What's the initial username and password?

### DISCUSSION THREAD

**Discussion by songlinshu** [go-shiori/shiori#405] [Q&A] [answered]
hi,What's the initial username and password?
![image](https://user-images.githubusercontent.com/45619496/158287926-f6a5efc0-b4b2-4e0f-af3b-efe55127b5b3.png)

**Comment by andrewrkerr** [comment, marked as answer]
Linking to faq in case it changes:
https://github.com/go-shiori/shiori/wiki/Frequently-Asked-Question

Look under "What is the default account to login at the first time ?"

**Reply by Darthagnon** [reply]
wiki is now 404, documentation [here](https://github.com/go-shiori/shiori/blob/master/docs/Usage.md#using-web-interface), which says the default is:
```
username: shiori
password: gopher
```
but I'm not having any luck, even with that. You need to run the `shiori migrate` command to create the database tables before you can log in. Seems to be undocumented, except in the Github releases notes @fmartingr

**Reply by fmartingr** [reply]
Hey @Darthagnon, this was documented over the [1.5.3](https://github.com/go-shiori/shiori/releases/tag/v1.5.3) release notes, though fear not, I've just released [1.5.4](https://github.com/go-shiori/shiori/releases/tag/v1.5.4) which among other things, removes manual migrations. If you want to check the discussion about this, go ahead to #473.
